api/tests/unit/features/test_unit_features_views.py

```diff
@@ -2229,6 +2230,110 @@ def test_cannot_update_feature_of_a_feature_state(
     )
 
 
+@pytest.mark.parametrize(
+    "client",
+    [lazy_fixture("admin_master_api_key_client"), lazy_fixture("admin_client")],
+)
+def test_create_feature_with_required_metadata_returns_400(
+    project: Project,
+    client: APIClient,
+    required_a_segment_metadata_field,
+):
```

Comment:
```suggestion
) -> None:
```

(also applies to tests below)

Reply:
Added.